src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -875,7 +875,7 @@ def makeSubmit(self, jobList):
             jdl.append("+JobLeaseDuration = isUndefined(MachineAttrMaxHibernateTime0) ? 1200 : MachineAttrMaxHibernateTime0\n")
 
             ### print all the variables needed for us to rely on condor userlog
-            jdl.append("job_ad_information_attrs = JobStatus,QDate,EnteredCurrentStatus,JobStartDate,DESIRED_Sites,ExtDESIRED_Sites,WMAgent_JobID,MATCH_EXP_JOBGLIDEIN_CMSSite\n")
+            jdl.append("job_ad_information_attrs = JobStatus,QDate,EnteredCurrentStatus,DESIRED_Sites,ExtDESIRED_Sites,WMAgent_JobID,MATCH_EXP_JOBGLIDEIN_CMSSite\n")
```

Comment:
Almost.  

For running jobs, `MATCH_EXP_JOBGLIDEIN_CMSSite` should show up in the condor log if the job was previously run and is being restarted a second time; I believe `MATCH_EX_JOBGLIDEIN_CMSSite` should show the site where the job previously run.

`JOBGLIDEIN_CMSSite` should point at the correct site for running jobs.

Since the job is not necessarily running when this code executes, maybe this is the most bulletproof variant:
```
if ad.get("JOBGLIDEIN_CMSSite", "").startswith("$$"): #JOBGLIDEIN_CMSSite present but not evaluated
  site = ad.get("MATCH_EXP_JOBGLIDEIN_CMSSite", "Unknown")
else:
  site = ad.get("JOBGLIDEIN_CMSSite", "Unknown")
```